lib/Chainable.js

```diff
@@ -1,8 +1,16 @@
 'use strict';
 
+const IDENTIFIER = '@@chained-config/Chainable';
+
 class Chainable {
     constructor(parent) {
         this.parent = parent;
+
+        Object.defineProperty(this, IDENTIFIER, { value: true });
+    }
+
+    static isChainable(value) {
```

Comment:
Should we drink our own Kool-aid and use [js-class-is](https://github.com/moxystudio/js-class-is) for this? 😉

Reply:
Ahaha will do.

Reply:
@fsdiogo when will we be landing 1.0.0?

Reply:
I've discovered a bug in `js-class-is`. Made a PR:  https://github.com/moxystudio/js-class-is/pull/6

Reply:
The MR was merged! I updated the version and tests are now passing.